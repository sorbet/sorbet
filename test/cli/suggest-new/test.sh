#!/bin/sh
#!/bin/bash
main/sorbet --silence-dev-message test/cli/suggest-new/suggest-new.rb 2>&1
