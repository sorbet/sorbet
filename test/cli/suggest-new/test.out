test/cli/suggest-new/suggest-new.rb:9: Method `ABC` does not exist on `T.class_of(<root>)` https://srb.help/7003
     9 |ABC(1)
        ^^^
  Note:
    Ruby uses `.new` to invoke a class's constructor
  Autocorrect: Use `-a` to autocorrect
    test/cli/suggest-new/suggest-new.rb:9: Insert `.new`
     9 |ABC(1)
           ^
  Note:
    Ruby uses `.new` to invoke a class's constructor
  Autocorrect: Use `-a` to autocorrect
    test/cli/suggest-new/suggest-new.rb:9: Insert `.new`
     9 |ABC(1)
           ^
Errors: 1
